src/bsp/src/esp/src/UserInterface.cpp

```diff
@@ -0,0 +1,42 @@
+#include "UserInterface.h"
+#include "esp_log.h"
+#include <cstdio>
+
+int UserInterface::print(const char* format, va_list args) const {
+    int ret = vprintf(format, args);
+    printf("\n\r");
+    return ret;
+}
+
+int UserInterface::print(const char* format, ...) const {
+    va_list args;
+    va_start(args, format);
+    int retValue = print(format, args);
+    va_end(args);
+
+    return retValue;
+}
+
+int UserInterface::logInfo(const char* format, va_list args) const {
+    esp_log_writev(ESP_LOG_INFO, "Logger", format, args);
```

Comment:
What does "Logger" stands for, it's a prefix?

Reply:
ESP wants a tag for its log writing. I think the goal is to set different log levels to different tags. I will change it to use vprintf to have a usefull return type